shell/shell.c

```diff
@@ -897,6 +897,7 @@ static void set_view_type(ShellViewType viewtype, gboolean reload)
 
     shell->normalize_percentage = TRUE;
     shell->view_type = viewtype;
+    shell->_order_type = 0;
```

Comment:
The % is inverted later if shell->_order_type = 1.
https://github.com/bp0/hardinfo/blob/2e585991f0880a8a6596c65a364c622d28b24961/shell/shell.c#L1245